kits/flutter_ume_kit_dio/lib/src/widgets/pluggable_state.dart

```diff
@@ -48,11 +48,13 @@ class DioPluggableState extends State<DioInspector> {
   /// Using [setState] won't cause too much performance regression,
   /// since we've implemented the list with `findChildIndexCallback`.
   void _listener() {
-    if (mounted &&
-        !context.debugDoingBuild &&
-        context.owner?.debugBuilding != true) {
-      setState(() {});
-    }
+    Future.microtask(() {
+      if (mounted &&
+          !context.debugDoingBuild &&
+          context.owner?.debugBuilding != true) {
```

Comment:
```suggestion
          !context.debugDoingBuild &&
          !context.owner?.debugBuilding) {
```

It seems better to unify the two styles of judging non-value.

Reply:
😂 Again, you can't use `!` on a nullable field.

Reply:
😂